src/map/utils/battleutils.cpp

```diff
@@ -2871,15 +2871,13 @@ namespace battleutils
         // https://www.bg-wiki.com/bg/Critical_Hit_Rate
         int32 attackerAgi = PAttacker->AGI();
         int32 defenderAgi = PDefender->AGI();
-        int32 dAGI        = attackerAgi - defenderAgi;
-        int32 dAgiAbs     = std::abs(dAGI);
+        int32 dAgi        = attackerAgi - defenderAgi;
+        // Only consider positive dAgi
+        int32 dAgiPositive = std::max(0, dAgi);
 
-        // Default to +0 crit rate
-        int32 critRate = 0;
-
-        critRate = dAgiAbs / 10;
+        int32 critRate = dAgiPositive / 10;
 
-        return std::min(critRate, 15);
+        return critRate;
```

Comment:
Doesnt this remove the 15 cap?

Reply:
Comment should be updated. However, I think the cap should remain until proven otherwise. Just my opinion though.

Reply:
I believe it does say there is a cap unless I am misunderstanding this line:

`you can see that there is an extremely significant jump in critical hit rate when dDEX is increased from 40 to 50 (from 4% at dDEX=39 to 15% at dDEX=50), but no further boost to critical hit rate once dDEX is greater than 50.`

Reply:
> I believe it does say there is a cap unless I am misunderstanding this line:
> 
> `you can see that there is an extremely significant jump in critical hit rate when dDEX is increased from 40 to 50 (from 4% at dDEX=39 to 15% at dDEX=50), but no further boost to critical hit rate once dDEX is greater than 50.`

That is for dDex to normal crits, not dAgi to ranged crit hits. This PR only deals with ranged crit hits.